engine.py

```diff
@@ -364,3 +358,20 @@ def _run_app_instance_commands(self):
         # finally, run the commands
         for command in commands_to_run:
             command()
+
+    #####################################################################################
+    # Logging
+
+    def _emit_log_message(self, handler, record):
+        """
+        Called by the engine to log messages in Alias Terminal.
+        All log messages from the toolkit logging namespace will be passed to this method.
+
+        :param handler: Log handler that this message was dispatched from.
+                        Its default format is "[levelname basename] message".
+        :type handler: :class:`~python.logging.LogHandler`
+        :param record: Standard python logging record.
+        :type record: :class:`~python.logging.LogRecord`
+        """
+        # TODO: improve Alias API to redirect the logs to the Alias Promptline
```

Comment:
Logged https://jira.autodesk.com/browse/SHOT-3371 for this

Reply:
hey @rob-aitchison , there is already a ticket for this TODO item :)
https://jira.autodesk.com/projects/SHOT/issues/SHOT-3361

Reply:
Haha - cool! :) I linked it as a duplicate.